tasks/setup_wp_databases.yml

```diff
@@ -23,39 +23,45 @@
   register: wp_tables
   changed_when: false
 
-- name: Create SQL tempfile
-  tempfile:
-    state: file
-    suffix: .wordpress.sql
-  register: wp_sql_file
+- block:
+  - name: Create SQL tempfile
+    tempfile:
+      state: file
+      suffix: .wordpress.sql
+    register: wp_sql_file
+
+  - name: Create new WordPress SQL file
+    template:
+      src: templates/wordpress.sql.j2
+      dest: "{{ wp_sql_file.path }}"
+    when: not wordpress_import_db_file
+
+  - name: Copy WordPress SQL file from localhost
+    copy:
+      src: "{{ inventory_hostname }}/wordpress.sql"
+      dest: "{{ wp_sql_file.path }}"
+    when: wordpress_import_db_file
+
+  - name: Replace old URLs
+    replace:
+      path: "{{ wp_sql_file.path }}"
+      regexp: "{{ item }}"
+      replace: "{{ wordpress_siteurl }}"
+    loop: "{{ wordpress_import_replace_siteurls }}"
+    when: wordpress_import_db_file
+
+  - name: Import WordPress SQL file
+    mysql_db:
+      state: import
+      name: "{{ wordpress_mysql_database_name }}"
+      target: "{{ wp_sql_file.path }}"
```

Comment:
I think you forgot the condition here

Reply:
The condition is in line 64 and applies to the whole block. Previously, it was repeated for each task.

Reply:
please rereview @dasJ